v21.1/performance-recipes-solutions.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Performance tuning recipe solutions
+summary: Identifying possible fixes for common performance problems
+toc: true
+toc_not_nested: true
+---
+
+This page provides solutions for common performance issues in your clusters. See [the recipes](performance-recipes.html) to identify performance problems in your workload.
+
+<div class="filters clearfix">
+    <button class="filter-button page-level" data-scope="contention"><strong>Contention</strong></button>
+    <button class="filter-button page-level" data-scope="fullscans">Full table scans<strong></strong></button>
+    <button class="filter-button page-level" data-scope="primarykey">Suboptimal primary keys<strong></strong></button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="contention">
+## Indicators that your workoad is experiencing contention
+
+Your application is experiencing degraded performance with serialization errors like:
+
+  - `SQLSTATE: 40001`
+  - `RETRY_WRITE_TOO_OLD`
+  - `RETRY_SERIALIZABLE`
+
+The [SQL Statement Contention graph](ui-sql-dashboard.html#sql-statement-contention) graph is showing spikes over time.
+
+{%comment%} Add screenshot of high-contention graph {%endcomment%}
+
+The [SQL Statement Errors graph](ui-sql-dashboard.html#sql-statement-errors) graph is showing spikes in retries over time.
```

Comment:
This should be the Transaction Restarts chart which is now on the DB console SQL metrics page (used to be on the Distributed metrics dashboard). 

https://github.com/cockroachdb/cockroach/commit/ef5135df0e514552eee99bad7264132508cf3f41